docs/configure.md

```diff
@@ -0,0 +1,279 @@
+# Create a launch configuration
+
+In Visual Studio Code, the `launch.json` configuration file connects a debug session to the target hardware via a debug
+adapter. The following explains how to create the file for single- and multi-core devices using different debug adapters.
+
+The pseudo debugger types `cmsis-debug-pyocd` and `cmsis-debug-jlink` are available to set up the debug connection in the
+`launch.json` configuration file. However, these are not full debug adapters but generate debug configurations of type
+`gdbtarget` which comes with the
+[CDT GDB Debug Adapter Extension](https://marketplace.visualstudio.com/items?itemName=eclipse-cdt.cdt-gdb-vscode) that is
+part of the Arm CMSIS Debugger extension pack.
+
+!!! Note
+    Using pyOCD, you can connect to the target via CMSIS-DAP and ST-LINK debug probes.
+
+## Prerequisites
+
+Make sure you have made the [required changes](./setup.md#project-setup) to your CMSIS Solution-based project.
+
+If your project does not yet contain a `launch.json` file, create it as follows:
```

Comment:
There is a simpler procedure to achieve that:
* Go to `Run and Debug` view
![image](https://github.com/user-attachments/assets/310d8320-e943-492f-9f63-8dc48ec80710)
* Click `create a launch.json file` link in the text. A quick-pick shows
![image](https://github.com/user-attachments/assets/c8044939-0096-4f5d-bd99-bca6cbd1a02d)
* Select `CMSIS Debugger (pyOCD)` or `CMSIS Debugger (J-LINK)`
* This creates a new launch.json file under `.vscode` pre-filled with the selected default configuration.

Reply:
I have made the required changes.